Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM rastasheep/ubuntu-sshd
+
+RUN locale-gen en_US.UTF-8
+
+ENV LC_ALL="en_US.UTF-8"
+ENV LANG="en_US.UTF-8"
+ENV LANGUAGE="en_US.UTF-8"
+
+RUN apt-get update && apt-get -y install sudo tree
+
+# Add bolt user
+RUN useradd bolt && echo "bolt:bolt" | chpasswd && adduser bolt sudo
+
+# Add SSH key support
+RUN mkdir -p /home/bolt/.ssh/
+COPY spec/fixtures/keys/id_rsa.pub /home/bolt/.ssh/id_rsa.pub
+COPY spec/fixtures/keys/id_rsa.pub /home/bolt/.ssh/authorized_keys
+RUN touch /home/bolt/.ssh/known_hosts
+RUN echo "    IdentityFile /home/bolt/.ssh/id_rsa" >> /etc/ssh/ssh_config
```

Comment:
I think the test below will fail if you remove this line. We expect to require insecure in tests.